projects/systelab-components/src/lib/modal/message-popup/message-popup.service.ts

```diff
@@ -15,10 +16,14 @@ export class MessagePopupService {
 
 	public static readonly breakpointMedium = 500;
 
-	constructor(protected i18nService: I18nService, protected dialogService: DialogService) {
+	constructor(protected i18nService: I18nService, protected dialogService: DialogService, private overlayDispatcher: OverlayKeyboardDispatcher) {
```

Comment:
do you need this to be overlayKeyboard dispatcher? I have seen  there is BaseOverlayDispatcher more generic, and keyboard if you want to manage events on keyboard too

Reply:
I didn't know, but I tried and BaseOverlayDispatcher is not accesible,, not exported in the class, is used only to extend. Only it can be used OverlayKeyboardDispatcher or OverlayOutsideClickDispatcher